cli_reference/openshift_developer_cli/creating-a-project-in-odo.adoc

```diff
@@ -0,0 +1,31 @@
+[id="creating-a-project"]
+= Creating a project
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: configuring-odo-cli
+toc::[]
+
+You can use {odo-title} to create a project.
+
+.Procedure
+
+. Log in to an {product-title} cluster:
++
+----
+$ odo login -u developer -p developer
+----
+
+. Create a new directory for your project and make it your current directory:
```

Comment:
This is incorrect as within `odo` we do not commonly create a folder for a project, but rather for components.

I'd be more correct to say: `Create a new directory for your component(s)`

```sh
$ mkdir my_components
$ cd my_components
```